core/domain/question_services.py

```diff
@@ -712,3 +714,61 @@ def get_interaction_id_for_question(question_id):
     if question is None:
         raise Exception('No questions exists with the given question id.')
     return question.question_state_data.interaction.id
+
+
+def untag_deleted_misconceptions(committer_id, skill_id, skill_description):
```

Comment:
I was expecting this method to be called with `deleted_skill_misconception_ids` instead of this method internally fetching the skill and assuming that there's some deleted misconception ids which need to be deleted. (Also, `deleted_skill_misconception_ids` param will help us to some isolate question services with skill services.)